### PR TITLE
Added files All Assignments and assesment 1 and Assesment 2... 

### PR DESCRIPTION
**Pull request by shivnanda6272**

### CODE REVIEW

File: Evaluation/Eval_28Nov20/ShivnandaE1.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            <input type="text" id="Lname" name="fname" value="" placeholder="Last Name">
+            <br><br>
+            <label for="phone"><b> Mobile Number &nbsp;:&nbsp;&nbsp; </b></label>
+            <input type="tel" id="phone" name="phone" value="" placeholder="Mobile Number" pattern="[0-9]{10}">
+            <br><br>
+            <label for="add"><b> Address   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;:&nbsp;&nbsp; </b></label>
+            <input type="text" id="add" name="add" value="" placeholder="Address">
+            <br><br>
+            <label for="email"><b> Email &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;:&nbsp;&nbsp; </b></label>
+            <input type="tel" id="email" name="email" value="" placeholder="Email">
```
Comment (Snehal-113) [inline Evaluation/Eval_28Nov20/ShivnandaE1.html]: input type="email"

**Review by Snehal-113 — COMMENTED** [review]
Good!

File: Assignments/HTML_CSS/Assign1/ShivnandaA1/ShivnandaA1A4.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<html>
+    <head>
+        <title> About Myself </title>
+    </head>
+    <body style="background-color: blueviolet;">
+        <h1 style="color: cornsilk; text-align: center;">  MYSELF </h1>
+        <h2>           NAME : SHIVNANDA TELANGE  </h2>
+        <h3><p>        I am basically from Nanded but currently staying in Pune. <br> I have done my graduation in BSC computer science and pursuing MSC computer science in Dr. d y patil college. </p></h3>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign1/ShivnandaA1/ShivnandaA1A4.html]: never use p tag in heading tag

File: Assignments/HTML_CSS/Assign2/ShivnandaA2/ShivnandaA2.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<html>
+    <head>
+        <title> Assignment 2. </title>
+    </head>
+    <body style="background-color: deeppink;">
+        <h1 style="text-align: center;"><b> Google </b></h1>
+        <i> <small> This article is about company.</small> </i>
+        <h2><b> From the garage to the Googleplex </b></h2>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign2/ShivnandaA2/ShivnandaA2.html]: no need of b tag

File: Evaluation/Eval_28Nov20/ShivnandaE1.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        <h3> <em>User Details <em></h3>
+        <form>
+            <label for="Fname"><b> First Name  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;:&nbsp;&nbsp; </b></label>
+            <input type="text" id="Fname" name="fname" value="" placeholder="First Name"> 
+            <br><br>
+            <label for="Lname"><b> Last Name &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;:&nbsp;&nbsp; </b></label>
+            <input type="text" id="Lname" name="fname" value="" placeholder="Last Name">
+            <br><br>
+            <label for="phone"><b> Mobile Number &nbsp;:&nbsp;&nbsp; </b></label>
+            <input type="tel" id="phone" name="phone" value="" placeholder="Mobile Number" pattern="[0-9]{10}">
```
Comment (Pawan1005) [inline Evaluation/Eval_28Nov20/ShivnandaE1.html]: input type =number

File: Assignments/HTML_CSS/Assign7/ShivnandaA7/ShivnandaA7_TP.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        </tr>
+    </table>
+    </center>
+    <br><br>
+    <input type="submit" value="Next">
+    <input type="reset" value="Cancle">
+    </fieldset>
+    </form>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign7/ShivnandaA7/ShivnandaA7_TP.html]: add new line

File: Assignments/HTML_CSS/Assign7/ShivnandaA7/ShivnandaA7_FP.html [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        <input type="text" id="iFname" name="fname" value="" placeholder="First Name" required> 
+        <br><br>
+        <label for="imname"><b> Middle Name &nbsp;<sup style="color: darkred;">*</sup>&nbsp;&nbsp;: </b></label>
+        <input type="text" id="imname" name="imname" value="" placeholder="Middle Name" required>
+        <br><br>
+        <label for="iLname"><b> Last Name &nbsp;<sup style="color: darkred;">*</sup>&nbsp;&nbsp;: </b></label>
+        <input type="text" id="iLname" name="fname" value="" placeholder="Last Name" required>
+        <br><br>
+        <label for="mobile"><b> Mobile Number &nbsp;<sup style="color: darkred;">*</sup>&nbsp;&nbsp;: </b></label>
+        <input type="tel" id="mobile"name="mobile" placeholder="Mobile Number" required>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign7/ShivnandaA7/ShivnandaA7_FP.html]: type= number

File: Assignments/HTML_CSS/Assign7/ShivnandaA7/ShivnandaA7_5P.html [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+        </tr>
+        <tr>
+            <td>1</td>
+            <td> <input type="text" id="org" name="org"></td>
+            <td><input type="date" id="sdate" name="sdate"></td>
+            <td><input type="date" id="edate" name="edate"></td>
+            <td><input type="number" id="num" name="num"></td>
+        </tr>
+    </table>
+</center>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign7/ShivnandaA7/ShivnandaA7_5P.html]: we dont use center as a tag

File: Assignments/HTML_CSS/Assign7/ShivnandaA7/ShivnandaA7_5P.html [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+    <h1> Noukri</h1>
+    <form action="ShivnandaA7_home.html" autocomplete="on">
+    <fieldset class="abc">
+        <h3> Professional Details </h3>
+        <br><br>
+    <legend> Professional Details</legend>
+    <a href="ShivnandaA7_home.html" target="_top" style="color: white;"> Home |</a>
+        <a href="ShivnandaA7_login.html" target="_top"  style="color: white;"> Login | </a>
+        <a href="ShivnandaA7_FP.html" target="_top" style="color: white;"> New User </a><br> <br><br>
+    <center>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign7/ShivnandaA7/ShivnandaA7_5P.html]: center tag  older version

File: Assignments/HTML_CSS/Assign7/ShivnandaA7/NaukriS.css [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+    text-align: center;
+}
+.body_img
+{
+    background-image: url("SSS.jpeg");
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+
+}
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign7/ShivnandaA7/NaukriS.css]: add new line

File: Assignments/HTML_CSS/Assign7/ShivnandaA7/ShivnandaA7_TP.html (hunk 5 above)
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign7/ShivnandaA7/ShivnandaA7_TP.html]: add new line

File: Evaluation/Eval_28Nov20/ShivnandaE1.css [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+td,td{
+    border-style: solid;
+    padding: 30px;
+}
+th{
+    text-align: left;
+}
+.frnt_cls{
+    font-family: cursive;
+}
```
Comment (Pawan1005) [inline Evaluation/Eval_28Nov20/ShivnandaE1.css]: add new line

File: Assignments/HTML_CSS/Assign6/Shivnanda6/ShivnandaA6.html [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+            <label for="Male">Male</label>
+            <input type="radio" id="Female" name="Gender" value="Female">
+            <label for="Female">Female</label>
+            <input type="radio" id="other" name="Gender" value="other">
+            <label for="other">Other</label>
+            <br><br>
+            <label for="Age"><b> Age &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; :</b></label>
+            <input type="radio" id="Less" name="Age" value="Less">
+            <label for="Less">Less than 18</label>
+            <input type="radio" id="18_25" name="Age" value="18_25" checked>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign6/Shivnanda6/ShivnandaA6.html:29]: wrong attribute

File: Assignments/HTML_CSS/Assign6/Shivnanda6/ShivnandaA6.html [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+                <option value="DCS"> Diploma in computer science</option>
+                <option value="SG"> Science Graduate</option>
+                <option value="BE"> Bachelor in engg</option>
+                <option value="MCS"> Master in Computer Science</option>
+                <option value="ME"> Master in Engg</option>
+                <option value="other"> Other</option>
+            </select>
+            <br><br> 
+            <label for="Uni"> <b>University Name :</b> </label>
+            <input list="u" id="uni" name="uni"/>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign6/Shivnanda6/ShivnandaA6.html]: no need of closing tag

File: Assignments/HTML_CSS/Assign6/Shivnanda6/ShivnandaA6.css [hunk 13]
```diff
@@ -0,0 +1,10 @@
+.frmt_cls{
+    background-color:grey;
+    border-style: outset;
+    border-bottom-left-radius: 10px;
+    border-top-left-radius: 10px;
+}
+.fnt_cls{
+    font-family: Arial, Helvetica, sans-serif;
+    text-align: right;
+}
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign6/Shivnanda6/ShivnandaA6.css]: add new line

File: Assignments/HTML_CSS/Assign5/ShivnandaA5/ShivnandaA5Hobbies.html [hunk 14]
```diff
@@ -0,0 +1,13 @@
+<html>
+    <body style="background-color: orangered;">
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign5/ShivnandaA5/ShivnandaA5Hobbies.html:5]: head tag missing

File: Assignments/HTML_CSS/Assign5/ShivnandaA5/Ass5.css [hunk 15]
```diff
@@ -0,0 +1,10 @@
+              .frmt_head{
+                background-color: orange;
+                border-style: solid;
+                border-color: black;
+            }
+            .ul,li{
+                list-style: square;
+                color: green;
+            }
+
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign5/ShivnandaA5/Ass5.css:10]: add new line

File: Assignments/HTML_CSS/Assign1/ShivnandaA1/ShivnandaA1Q1.html [hunk 16]
```diff
@@ -0,0 +1,10 @@
+<html>
+    <head> 
+        <title> Question 1. </title>
+    </head>
+    <body>
+        <h1> HTML </h1>
+        <h3> Introduction </h3>
+        <p> HTML is standard markup langunge for Web Pages.</p>
+    </body>    
+</html>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign1/ShivnandaA1/ShivnandaA1Q1.html]: add new line

File: Assignments/HTML_CSS/Assign2/ShivnandaA2/ShivnandaA2.html [hunk 17]
```diff
@@ -0,0 +1,20 @@
...
+        <p> They called this search engine <mark> Backrup.</mark></p>
+        <hr>
+        <h2><b> Initial public offering </b></h2>
+        <p> Google's initial public offering (IPO) took place five years later, on 19<sup>th </sup> August 2004. <br> At that time Larry Page. Sergey Brin, and Eric Schmidt agreed to work together at Google for <del>30 </del> <ins>20</ins>years, until the year 2024.</p>
+        <marquee behavior="alternate">
+            <p style="text-align: center;color:darkgreen"> Thank You!!! </b>
+        </marquee>
+    </body>
+</html>
+
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign2/ShivnandaA2/ShivnandaA2.html]: add new line

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Pull request merged**